test/benchmark/README.md

```diff
@@ -0,0 +1,223 @@
+# Benchmark
+
+This benchmark focus on testing KFServing performance with and without Knative queue proxy/activator on the request path.
+
+* Knative queue proxy does the following for the KFServing main container.
+  - Enforces concurrency level for the pod
+  - Emit metrics for autoscaling(KPA)
+  - Timeout enforcement
+  - Readiness probe
+  - Queue limiting
+  - Distributed tracing
+  - Graceful shutdown handling
+
+* Knative activator buffers the requests while pods are scaled down to zero and report metrics to autoscaler. The activator
+also effectively acts as a load balancer which distributes the load across all the pods as they become available in a way that
+does not overload them with regards to their concurrency settings. So it protects the app from burst so you do not see messages
+queuing in the user pods.
+
+## Environment Setup
+- K8S: v1.14.10-gke.36
+- Istio: 1.1.6
+- Knative: 0.11.2
+- KFServing: master
+
```

Comment:
Can we say what GKE or other cluster nodes were used so easier for people to replicate?

Reply:
ye I mentioned right above it is `v1.14.10-gke.36`

Reply:
So all defaults - 3 node n1-standard? Would it be possible to run node affinity to make the model run on a separate node to vegeta?

Reply:
it is 8 nodes n1-standard, the vegeta is run as a kubernetes job so you can specify node anti-affinity to run on a different node.